TicTacToe.py

```diff
@@ -229,13 +229,13 @@ def advancedGameSettings(self, settingToChange):
 			case 'change icons':
 				print('Both player icons must only be 1 character long.')
 				player0 = 'too long'
-				while len(player0) != 1:
+				while len(player0) != 1 or player0 == self.BLANK_POS_ICON:
 					player0 = input('What do you want first move icon to be? (Traditionaly X): ')
 					if len(player0) != 1:
 						print("Please enter a single character for the player icon")
 
 				player1 = 'too long'
-				while len(player1) != 1:
+				while len(player1) != 1 or player0 == self.BLANK_POS_ICON:
```

Comment:
What is the point of `or player0 == self.BLANK_POS_ICON`

Reply:
Prevents the user from assigning player icons to be the BLANK_POS_ICON, regardless of what the BLANK_POS_ICON is.

Reply:
This will be less necessary if we eventually change the backend for icons to be multiple characters long, but I think it's still a good check to have.

I also think I'm gonna do a full documentation run here in a bit, a comment might have cleared that up to begin with 😅

Reply:
This will become unnecessary with the completion of #45 as this whole function will have to be reworked.

Reply:
Can we push this through for now and it fixes the bug?  Doesn't change the amount of work that needs to be done later, and this work is already done.

Reply:
Fair enough